modules/Date.js

```diff
@@ -0,0 +1,13 @@
+import { DateTime } from './luxoncode.js';
+
+function currentTime() {
```

Comment:
- [ ] We want you to adhere to using _arrow_ syntax for functions in this activity. Kindly ensure to improve upon this aspect.